[2022-04-29 02:22:37,565] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-04-29 02:22:37,612] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-04-29 02:22:37,614] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 02:22:37,615] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-29 02:22:37,616] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 02:22:37,659] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): rm_files_task> on 2020-01-01 00:00:00+00:00
[2022-04-29 02:22:37,668] {standard_task_runner.py:52} INFO - Started process 3596 to run task
[2022-04-29 02:22:37,676] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'rm_files_task', 'scheduled__2020-01-01T00:00:00+00:00', '--job-id', '644', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmpdegeegwg', '--error-file', '/tmp/tmpu3s6hxwy']
[2022-04-29 02:22:37,678] {standard_task_runner.py:80} INFO - Job 644: Subtask rm_files_task
[2022-04-29 02:22:38,074] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2020-01-01T00:00:00+00:00 [running]> on host 6fd0ba274d58
[2022-04-29 02:22:38,441] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-29 02:22:38,628] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=rm_files_task
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T00:00:00+00:00
[2022-04-29 02:22:38,635] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-29 02:22:38,638] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/laucnty20.xlsx /opt/***/us_employment_2020.parquet']
[2022-04-29 02:22:38,657] {subprocess.py:85} INFO - Output:
[2022-04-29 02:22:38,740] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-29 02:22:38,959] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=rm_files_task, execution_date=20200101T000000, start_date=20220429T022237, end_date=20220429T022238
[2022-04-29 02:22:39,022] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-29 02:22:39,280] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-29 02:41:03,219] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-04-29 02:41:03,269] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2020-01-01T00:00:00+00:00 [queued]>
[2022-04-29 02:41:03,271] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 02:41:03,272] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-29 02:41:03,273] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-29 02:41:03,323] {taskinstance.py:1270} INFO - Executing <Task(BashOperator): rm_files_task> on 2020-01-01 00:00:00+00:00
[2022-04-29 02:41:03,335] {standard_task_runner.py:52} INFO - Started process 6678 to run task
[2022-04-29 02:41:03,355] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcp_dag', 'rm_files_task', 'scheduled__2020-01-01T00:00:00+00:00', '--job-id', '789', '--raw', '--subdir', 'DAGS_FOLDER/ingest_to_gcs_dag.py', '--cfg-path', '/tmp/tmps5are235', '--error-file', '/tmp/tmpgvqxol6g']
[2022-04-29 02:41:03,365] {standard_task_runner.py:80} INFO - Job 789: Subtask rm_files_task
[2022-04-29 02:41:03,763] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcp_dag.rm_files_task scheduled__2020-01-01T00:00:00+00:00 [running]> on host 6fd0ba274d58
[2022-04-29 02:41:04,026] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:156 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-04-29 02:41:04,217] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcp_dag
AIRFLOW_CTX_TASK_ID=rm_files_task
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T00:00:00+00:00
[2022-04-29 02:41:04,222] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-29 02:41:04,225] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/laucnty20.xlsx /opt/***/us_employment_2020.parquet']
[2022-04-29 02:41:04,245] {subprocess.py:85} INFO - Output:
[2022-04-29 02:41:04,313] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-29 02:41:04,525] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcp_dag, task_id=rm_files_task, execution_date=20200101T000000, start_date=20220429T024103, end_date=20220429T024104
[2022-04-29 02:41:04,601] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-29 02:41:04,836] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
